bubblewrap.c

```diff
@@ -3276,6 +3282,30 @@ main (int    argc,
       setsid () == (pid_t) -1)
     die_with_error ("setsid");
 
+  if (opt_foreground) {
+    cleanup_fd int tty_fd = -1;
+
+    if (setpgid (0, 0) == -1)
+      die_with_error ("setpgid");
+
+    tty_fd = open ("/dev/tty", O_RDONLY|O_CLOEXEC);
+    if (tty_fd >= 0) {
+      sigset_t mask, old_mask;
+
+      sigemptyset (&mask);
+      sigaddset (&mask, SIGTTOU);
+
+      if (sigprocmask (SIG_BLOCK, &mask, &old_mask) == -1)
+        die_with_error ("sigprocmask");
+
+      if (tcsetpgrp (tty_fd, getpgid(0)) == -1)
```

Comment:
Space before opening parenthesis:

```suggestion
      if (tcsetpgrp (tty_fd, getpgid (0)) == -1)
```